docs/user/grading.rst

```diff
@@ -8,6 +10,15 @@ Grading Keys can be either:
 - ``Grade`` type which maps the points to a letter or numeric grade with a ``First Passing Grade``. This can be used e.g., for final exams or courses without exams that are graded solely by exercises.
 - ``Bonus`` type which maps the points to a numeric value. This can be used e.g., when the grade obtained in the given course or exam is not an end result but complements a final exam.
 
+There are two configurable special grades that are automatically treated as failing grades:
+
+- ``Plagiarism Grade`` (default ``U``) is assigned when a student has received a :ref:`Plagiarism verdict <plagiarism_instructors>` in one or more exercises.
+- ``No-participation Grade`` (default ``X``) is assigned according to the conditions below for courses and exams:
+
+    - For a course, a student receives this grade if they do not start any exercise i.e., number of participations is 0.
+    - For an exam, a student receives this grade if they neither start nor submit the exam.
```

Comment:
Not starting the exam already implies not submitting. So not submitting should be enough as condition, shouldn't it?

Reply:
Makes sense, changed at https://github.com/ls1intum/Artemis/pull/6087/commits/d0e0144c3cc1a4c59481bbcee0bb93beb9995eaf